setup.py

```diff
@@ -29,7 +29,7 @@
     "pytest-notebook",
     "pytest-xdist",
     "pytype",
-    "ray[debug,tune]~=0.8.5",
+    "ray[debug,tune]>=0.8.5",
```

Comment:
Heh, upgrading Ray was one of the things I was going to ask you to look into. Good this just works, I thought `src/imitation/scripts/parallel.py` might need changes. If we're going to upgrade it let's specify >= the latest release (1.13.0), 0.84 is quite ancient now.

Reply:
Now specifies `>=1.13.0`